api/src/game.ts

```diff
@@ -56,13 +56,17 @@ const initializePlayers = () => {
 };
 
 export const switchToNextPlayer = (
-  actualPlayer: Player,
-  players: Array<Player>,
-): Player => {
-  if (actualPlayer.playerNumber === 1) return players[1];
-  return players[0];
+  actualPlayerID: number,
+): number => {
+  if (actualPlayerID === 1) return 2;
```

Comment:
In the game state you used `currentPlayerId` so in this case you should name this params `currentPlayerId`